### DISCUSSION TITLE
Option to hide .install Package if there's a metapackage

### DISCUSSION THREAD

**Discussion by stefan-brun** [chocolatey/ChocolateyGUI#942] [Ideas]
Hi,

it would be nice to have an option in the GUI to hide .install Packages if there's a Metapackage.

example Notepad++
![image](https://user-images.githubusercontent.com/42066027/169307942-d45c0cde-35ba-4ee1-830c-ca8a7106661e.png)

**Comment by QuimaxW** [comment]
Yes!

At the moment, I made a separate repository on our Nexus server and put the .install packages there. Really, only to not confuse end-users.

**Comment by gep13** [comment]
I thought we had an issue around this, but I have just checked, and it doesn't look like we do.  Having the ability to hide/filter certain packages out of the Chocolatey GUI interface is something that has been requested before, but looks like this wasn't captured as an issue (if one of you wanted to raise that as an issue, filling out the details of the issue template, that would be great).
